assets/src/scripts/store/index.js

```diff
@@ -161,18 +161,18 @@ export const Actions = {
         };
     },
 
-    retrieveCustomTimeSeries(site, startTime, endTime) {
+    retrieveCustomTimeSeries(site, startTime, endTime, parmCd) {
```

Comment:
Could you add one test that tests that the passed in parmCd is used rather than the parmCd from the Redux state?